src/k8s-engine/wsl.ts

```diff
@@ -873,6 +875,41 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     const nerdctlPath = await this.wslify(windowsNerdctlPath);
 
     await this.runInstallScript(INSTALL_WSL_HELPERS_SCRIPT, 'install-wsl-helpers', nerdctlPath);
+    await this.installCredentialHelper();
+  }
+
+  protected async getHostIPAddr(): Promise<string> {
```

Comment:
How is this different from [`wslHostIPv4Address`](https://github.com/rancher-sandbox/rancher-desktop/blob/12881b8584365ff9694a928e49872fd2d5005a73/src/utils/networks.ts#L3)?

Reply:
It's older :)